setup.sh

```diff
@@ -37,6 +37,11 @@ nethermindImage=$NETHERMIND_IMAGE
 rethImage=$RETH_IMAGE
 mevBoostImage=$MEV_BOOST_IMAGE
 
+if [[ "$dataDir" == /* ]]; then
+else
+  pathPrefix=$currentDir/
```

Comment:
why don't just modify datadir here and rest nothing else needs to change

Reply:
I think this should fix it.